crates/net/network/src/transactions/mod.rs

```diff
@@ -249,7 +249,7 @@ pub struct TransactionsManager<
     N: NetworkPrimitives = EthNetworkPrimitives,
     PBundle: TransactionPolicies = NetworkPolicies<
         TransactionPropagationKind,
-        StrictEthAnnouncementFilter,
+        ShardedMempoolAnnouncementFilter,
```

Comment:
this then can be `ShardedMempoolAnnouncementFilter<StrictEthAnnouncementFilter>`